lightstep-tracer-jre/src/main/java/com/lightstep/tracer/jre/Version.java

```diff
@@ -0,0 +1,5 @@
+package com.lightstep.tracer.jre;
+
+class Version {
+    static final String LIGHTSTEP_TRACER_VERSION = "0.12.5";
```

Comment:
It should match what is in the pom.xml, which is 0.12.5. Running inc-version should modify it to 0.12.6